CommunicationLayer/CommDeviceControl/UdpMessageForwarder.cpp

```diff
@@ -43,10 +47,6 @@ UdpMessageForwarder::~UdpMessageForwarder()
 
 void UdpMessageForwarder::forwardData(QByteArray data)
 {
-    qDebug() << "UdpMessageForwarder: Forwarding data";
-    const quint64 dataWritten = socket_.writeDatagram(data, ipAddress_, port_);
-    if (dataWritten != static_cast<quint64>(data.size()))
-    {
-        qWarning() << "UdpMessageForwader: Unable to forward data";
```

Comment:
I did some reading on AmqpException and the subclasses of this and believe that what I've now implemented is the most appropriate way of dealing with the exceptions. I also created a neat little way of catching exceptions when creating the channel if the broker is down, and repeatedly trying to connect every few seconds for a couple minutes. Unfortunately after testing it it would seem that declaring a channel already has some form of wait in it, as the execution hangs waiting for the broker if it's not accessible. However it should be noted when I started up the broker the execution did not start in the few minutes I waited for it.